ckanext/depositar_theme/templates/home/index/idx_header.html

```diff
@@ -0,0 +1,139 @@
+            {% block header_account_notlogged %}
+            <li>{% link_for _('Log in'), named_route='user.login' %}</li>
+            {% if h.check_access('user_create') %}
+            <li>{% link_for _('Register'), named_route='user.register', class_='sub' %}</li>
+            {% endif %}
+            {{ language_selector() }}
+            {% endblock %}
+          </ul>
+        </nav>
+        {% endif %} {% endblock %}
+      </div>
+    </header>
+  {% endblock %}
+  <header class="navbar navbar-static-top masthead">
+    {% block header_debug %} {% if g.debug and not g.debug_supress_header %}
+    <div class="debug">Controller : {{ c.controller }}<br/>Action : {{ c.action }}</div>
+    {% endif %} {% endblock %}
+    <div class="container navbar-expand-md">
+      <div class="navbar-right">
+        <button data-target="#main-navigation-toggle" data-toggle="collapse" class="navbar-toggler collapsed" type="button" aria-label="expand or collapse" aria-expanded="false">
+          <span class="sr-only">{{ _('Toggle navigation') }}</span>
+          <span class="fa fa-bars"></span>
+        </button>
+      </div>
+      <hgroup class="{{ g.header_class }} navbar-left">
+        {% block header_logo %} {% if g.site_logo %}
+        <a class="logo" href="{{ h.url_for('home.index') }}"><img src="{{ h.url_for_static_or_external(g.site_logo) }}" alt="{{ g.site_title }}" title="{{ g.site_title }}" /></a>
+        {% else %}
+        <h1>
+          <a href="{{ h.url_for('home.index') }}">{{ g.site_title }}</a>
+        </h1>
+        {% if g.site_description %}
+        <h2>{{ g.site_description }}</h2>{% endif %} {% endif %} {% endblock %}
+      </hgroup>
+      
+      <div class="collapse navbar-collapse" id="main-navigation-toggle">
+        {% block header_site_navigation %}
+        <nav class="section navbar navigation">
+          <ul class="nav nav-pills">
+              {% block header_site_navigation_tabs %}
+                {{ h.build_nav_main(
+                  ('dataset.search', _('Datasets')),
+                  ('group.index', _('Topics')),
+                  ('organization.index', _('Projects')),
+                  ('home.about', _('About')),
+                  ('help.index', _('Help'))
+                ) }}
+              {% endblock %}
+          </ul>
+        </nav>
+        {% endblock %} {% block header_site_search %}
+        <form class="section site-search simple-input" action="{% url_for 'dataset.search' %}" method="get">
+          <div class="field">
+            <label for="field-sitewide-search">{% block header_site_search_label %}{{ _('Search Datasets') }}{% endblock %}</label>
+            <input id="field-sitewide-search" type="text" class="form-control" name="q" aria-label="{{ _('Search datasets') }}"/>
+            <button class="btn-search" type="submit" aria-label="{{ _('Submit') }}"><i class="fa fa-search"></i></button>
+          </div>
+        </form>
+        {% endblock %}
```

Comment:
覆蓋必要 block 即可，其他可以使用 super()

```suggestion
  {% block header_account %} {{ super() }} {% endblock %}
  <header class="navbar navbar-static-top masthead">
    {% block header_debug %} {{ super() }} {% endblock %}
    <div class="container navbar-expand-md">
      <div class="navbar-right">
        <button data-target="#main-navigation-toggle" data-toggle="collapse" class="navbar-toggler collapsed" type="button" aria-label="expand or collapse" aria-expanded="false">
          <span class="sr-only">{{ _('Toggle navigation') }}</span>
          <span class="fa fa-bars"></span>
        </button>
      </div>
      <hgroup class="{{ g.header_class }} navbar-left">
        {% block header_logo %} {{ super() }} {% endblock %}
      </hgroup>

      <div class="collapse navbar-collapse" id="main-navigation-toggle">
        {% block header_site_navigation %} {{ super() }} {% endblock %}
        {% block header_site_search %} {{ super() }} {% endblock %}
```